sound/soc/sof/intel/hda-codec.c

```diff
@@ -113,8 +114,14 @@ static int hda_codec_probe(struct snd_sof_dev *sdev, int address,
 	if (ret < 0)
 		return ret;
 
-	if ((resp & 0xFFFF0000) == IDISP_VID_INTEL)
+	if ((resp & 0xFFFF0000) == IDISP_VID_INTEL) {
+		if (!hda->i915_codec_enabled) {
+			dev_dbg(sdev->dev,
+				"iDisp hw present but no driver, ignoring\n");
```

Comment:
it's more 'will be ignored'. There is nothing in this function that ignores the codec, you just return a negative value that will be handled in hda_codec_probe_bus

Reply:
@plbossart hmm, I'll change this. but we do skip the codec driver probe later in the function, so first part of "ignoring" is already done in this function. but yeah, the info needs to be passed to controller side as well and that is done in the parent, so point taken.

Reply:
@plbossart "will be ignored" didn't fit 80-chars, so I left ", ignoring" part out.. not adding value